pkgs/applications/version-management/forgejo/default.nix

```
@@ -59,15 +76,25 @@ buildGoModule rec {
     "-X 'main.Tags=${lib.concatStringsSep " " tags}'"
   ];
 
+  preBuild = ''
+    go run build/merge-forgejo-locales.go
+  '';
+
   postInstall = ''
     mkdir $data
-    cp -R ./{public,templates,options} $data
+    cp -R ./{templates,options} ${frontend}/public $data
     mkdir -p $out
     cp -R ./options/locale $out/locale
     wrapProgram $out/bin/gitea \
       --prefix PATH : ${lib.makeBinPath [ bash git gzip openssh ]}
   '';
 
+  ## $data is not available in go-modules.drv and preBuild isn't needed
```

```suggestion
  # $data is not available in go-modules.drv and preBuild isn't needed
```

even if it isn't required, if it does nothing undesirable we should just leave it as is.

---

yeah, double `#` is a bit of a habit of mine :sweat_smile: 

will remove it :+1: